make `use-cases/privileged-access-management/` page

deliaconstantino opened the pull request:
Closes gravitational/web#2792
[Asana details](https://app.asana.com/0/1201922182803749/1201930473038991)
- preview [here](https://teleport-docs-next-git-delia-build-moderni-c02ee0-gravitational.vercel.app/use-cases/privileged-access-management/)

### Icon/Logo Updates
- update icons - needed to add `developer-workflow.svg` to `icons.ts`
- adds new company logos and creates new logo collection: `ModerizePam`
- used `card` icon for navbar dropdown:
<img width="550" alt="Screen Shot 2022-03-08 at 11 54 22 AM" src="https://user-images.githubusercontent.com/70108137/157286629-1e43a025-645a-4880-a5fb-e689c45b4fb3.png">

### `<PDFTeaser />` Updates
- adds `16px` `px` for `<PDFTeaser />` on desktop to match `ProductBanner` padding above this component; reverts to standard padding on mobile to match footer below
- adds additional `py` to `<PDFTeaser />` to match with rest of page
- adds optional `bg` prop to `<PDFTeaser />`
- update absolute positioning of shadow  image to be calculated from top of component, which maintains the same location with any amount of text.

alexwolfe commented:
@deliaconstantino  A few things
- PDF Image still appears to be broken in the preview not sure why
- You need a shadow on the image of the Teleport App
- Add the "Easy to get started" and "Try Teleport today" sections at the bottom of the page.

deliaconstantino commented:
> * here

Addressed these comments; here is the preview for them: https://teleport-docs-next-git-delia-build-moderni-c02ee0-gravitational.vercel.app/use-cases/privileged-access-management/

deliaconstantino commented:
per conversation with @C-STYR: 
- added an optional prop, `imageMarginLeft` to `ProductBanner` for full sized, right-aligned images that need space between image and text block
- resized gif to target 1488 x 800 pixels at 2x sizing. (final gif size: 1488 x 837 to keep its aspect ratio)

C-STYR commented on components/ProductBanner/ProductBanner.tsx:
```diff
@@ -111,6 +111,7 @@ export interface ProductBannerProps {
   imgPosition: "left" | "right";
   logoSrc?: string;
   videoSource?: string;
+  imageMarginLeft?: boolean;
```
I feel like this should have been padding rather than margin, thoughts? Happy to be convinced otherwise.
Also, if (and only if, please experiment) we're going to have this problem on a right-aligned full-size image too, let's make sure we account for that here and now.  Let's change the name of this prop to `addImagePadding` or "Margin", whatever we end up with.

deliaconstantino replied:
I tried padding first; however it does not work in this use case. Margin does successfully apply the needed space at all viewports.

Updated to `addImageMargin` which optionally adds `mx` at full screen sizes for right or left images

C-STYR commented on pages/use-cases/privileged-access-management/index.mdx:
```diff
@@ -0,0 +1,229 @@
+---
+
+import AnnouncementBar from "components/AnnouncementBar";
+import { ModerizePam } from "components/AnnouncementBar/data";
```
"ModerizePam" is misspelled here too.

deliaconstantino replied:
Thanks updated the spelling for all of these

C-STYR commented on components/AnnouncementBar/data.ts:
```diff
@@ -121,6 +121,26 @@ export const NewHomepage: LogoRow[] = expandConfig([
 ]);
 
+// use-cases/privileged-access-management page
+export const ModerizePam: LogoRow[] = expandConfig([
```
"ModerizePam" is misspelled. Missing an "n".

deliaconstantino replied:
updated

C-STYR commented on components/AnnouncementBar/logos.ts:
```diff
@@ -19,16 +19,19 @@ export { default as colorGitlab } from "./logos/color-gitlab.svg";
 export { default as colorGladly } from "./logos/color-gladly.svg";
 export { default as colorGSR } from "./logos/color-gsr.svg";
 export { default as colorGusto } from "./logos/color-gusto.svg";
+export { default as colorHealthTap } from "./logos/color-healthtap.svg";
```
I have not checked these logos for proper size, let's do that in our meeting today.

deliaconstantino replied:
updated size for vmware logo:
<img width="1142" alt="Screen Shot 2022-03-09 at 2 57 52 PM" src="https://user-images.githubusercontent.com/70108137/157523775-4b4a0128-256f-4537-92aa-2ea563d82307.png">

deliaconstantino replied:
It still looks like a slightly large gap, but I think that's because of the copyright symbol on left of vmware coupled with the slope of the Moody's "M". Adding these images for reference.
Before:
<img width="1021" alt="Screen Shot 2022-03-09 at 3 01 08 PM" src="https://user-images.githubusercontent.com/70108137/157524689-3b7d5e2e-fb35-44f5-9752-6fba585cef5c.png">

After:
<img width="1176" alt="Screen Shot 2022-03-09 at 3 00 25 PM" src="https://user-images.githubusercontent.com/70108137/157524712-2c3da473-a881-41e3-9b2d-6e5743f479a9.png">

C-STYR reviewed (changes requested):
Nice work.  Just a few bits to clean up, but overall a tidy job.
